client-admin/src/components/conversation-admin/conversation-config.js

```diff
@@ -284,64 +174,23 @@ class ConversationConfig extends React.Component {
           Schemes
         </Heading>
 
-        <Flex sx={{ alignItems: 'flex-start', mb: [3] }}>
-          <Box sx={{ flexShrink: 0, position: 'relative', top: -0.5 }}>
-            <input
-              type="checkbox"
-              data-test-id="strict_moderation"
-              ref={c => (this.strict_moderation = c)}
-              checked={this.props.zid_metadata.strict_moderation}
-              onChange={this.handleBoolValueChange('strict_moderation').bind(
-                this
-              )}
-            />
-          </Box>
-          <Box sx={{ ml: [2], flexShrink: 0, maxWidth: '35em' }}>
-            <Text>No comments shown without moderator approval</Text>
-          </Box>
-        </Flex>
-
-        <Flex sx={{ alignItems: 'flex-start', mb: [3] }}>
-          <Box sx={{ flexShrink: 0, position: 'relative', top: -0.5 }}>
-            <input
-              type="checkbox"
-              label="Require Auth to Comment"
-              data-test-id="auth_needed_to_write"
-              ref={c => (this.auth_needed_to_write = c)}
-              checked={this.props.zid_metadata.auth_needed_to_write}
-              onChange={this.handleBoolValueChange('auth_needed_to_write').bind(
-                this
-              )}
-            />
-          </Box>
-          <Box sx={{ ml: [2], flexShrink: 0, maxWidth: '35em' }}>
-            <Text>
-              Participants cannot submit comments without first connecting
-              either Facebook or Twitter
-            </Text>
-          </Box>
-        </Flex>
-
-        <Flex sx={{ alignItems: 'flex-start', mb: [3] }}>
-          <Box sx={{ flexShrink: 0, position: 'relative', top: -0.5 }}>
-            <input
-              type="checkbox"
-              label="Require Auth to Vote"
-              data-test-id="auth_needed_to_vote"
-              ref={c => (this.auth_needed_to_vote = c)}
-              checked={this.props.zid_metadata.auth_needed_to_vote}
-              onChange={this.handleBoolValueChange('auth_needed_to_vote').bind(
-                this
-              )}
-            />
-          </Box>
-          <Box sx={{ ml: [2], flexShrink: 0, maxWidth: '35em' }}>
-            <Text>
-              Participants cannot vote without first connecting either Facebook
-              or Twitter
-            </Text>
-          </Box>
-        </Flex>
+        <CheckboxField field="is_active" label="Conversation Is Open">
+          Conversation is open to participation
+        </CheckboxField>
+
+        <CheckboxField field="strict_moderation">
```

Comment:
This is the only checkbox without a label. Good time to add it? Any suggestion @pol-is/core?